gthockey-iOS/gthockey-iOS/AppDelegate.swift

```diff
@@ -19,21 +19,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                      didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
 
         self.window = UIWindow(frame: UIScreen.main.bounds)
-        let layout = UICollectionViewFlowLayout()
-        layout.sectionInset = UIEdgeInsets(top: 24.0, left: 0.0, bottom: 12.0, right: 0.0)
 
         let scheduleTabBarItem = UITabBarItem(title: "Schedule", image: UIImage(named: "ScheduleIcon"), tag: 0)
         let scheduleTableViewController = ScheduleTableViewController()
         let scheduleNavigationController = UINavigationController(rootViewController: scheduleTableViewController)
         scheduleNavigationController.tabBarItem = scheduleTabBarItem
-
+        
```

Comment:
Xcode -> Preferences -> Text Editing. Make sure both of these are checked off to eliminate the extra whitespace.
<img width="375" alt="Screen Shot 2019-10-06 at 8 38 45 PM" src="https://user-images.githubusercontent.com/13947356/66278674-608c3900-e879-11e9-88ce-459fdab50262.png">